collectors/python.d.plugin/anomalies/anomalies.chart.py

```diff
@@ -58,8 +58,7 @@ def __init__(self, configuration=None, name=None):
         self.collected_dims = {'probability': set(), 'anomaly': set()}
 
     def check(self):
-        python_version = float('{}.{}'.format(sys.version_info[0], sys.version_info[1]))
-        if python_version < 3.6:
+        if not (sys.version_info[0] >= 3 and sys.version_info[1] >= 6):
             self.error("anomalies collector only works with Python>=3.6")
```

Comment:
Can you please remind me why we don't `return False` here but just logging a warning message?